packages/css/src/utils/css-property.tsx

```diff
@@ -126,3 +128,17 @@ export function addUnitIfNeeded(
 
   return String(value).trim();
 }
+
+/**
+ * Returns `true` if the CSS property is defined within a template element
+ * (the string pieces of the template literal)
+ * rather than in the expression.
+ * Eg color: ${({ isPrimary }) => (isPrimary ? 'green' : 'red')};
+ * @param node
+ */
+export const isCssPropertyInTemplateElement = (node: t.TemplateElement): boolean => {
+  const re = /(\s.[a-z.\-_]*?\:)/; // `property:`
```

Comment:
Does this handle the case where the expression is on the next line? For example
```
color:
  ${() => ...};
```

Reply:
good point!
- [ ] check if this handle the case where the expression is on the next line

Reply:
Yep, it does handle that case. 
A similar output is generated in the `quasis` regardless of where the expression is on the next line. 
 
![image](https://user-images.githubusercontent.com/14813563/153782965-6988e16a-9fa2-4968-a4f1-1f6d47ab027e.png)